src/Raven.Server/NLog.default.config

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
are we going to copy this one or this is going to be embedded?

Reply:
copy